<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculadora</class>
 <widget class="QMainWindow" name="Calculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora de Hidralica e Pneumatica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>35</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_01">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>34</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QFrame" name="tela_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>531</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>O Diametro interno Ã©: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_02">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>CALCULAR</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>800</width>
       <height>150</height>
      </rect>
     </property>
     <widget class="QLabel" name="st_09">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_01">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>-10</y>
        <width>731</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>fundo_raiz.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="st_04">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_02">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>530</width>
        <height>10</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="perda">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>131</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>delta P</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_08">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>21</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pressao">
      <property name="geometry">
       <rect>
        <x>484</x>
        <y>90</y>
        <width>131</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>P</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>51</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>d</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>21</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="st_05">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>251</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1,6663785 * 10  *</string>
      </property>
     </widget>
     <widget class="QLabel" name="st_06">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>28</y>
        <width>16</width>
        <height>10</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vazao">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>113</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Q</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="comprimento">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>113</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Lt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="st_07">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>28</y>
        <width>29</width>
        <height>10</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1.85</string>
      </property>
     </widget>
     <widget class="QLabel" name="st_03">
      <property name="geometry">
       <rect>
        <x>579</x>
        <y>40</y>
        <width>21</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <zorder>st_01</zorder>
     <zorder>st_09</zorder>
     <zorder>st_04</zorder>
     <zorder>st_02</zorder>
     <zorder>perda</zorder>
     <zorder>st_08</zorder>
     <zorder>pressao</zorder>
     <zorder>st_11</zorder>
     <zorder>st_10</zorder>
     <zorder>st_05</zorder>
     <zorder>st_06</zorder>
     <zorder>vazao</zorder>
     <zorder>comprimento</zorder>
     <zorder>st_03</zorder>
     <zorder>st_07</zorder>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Q = VazÃ£o em  m  /h</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>30</y>
       <width>8</width>
       <height>8</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lt = Comprimento total da tubulaÃ§Ã£o em metros</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>delta P = Queda de pressÃ£o adimisivel em Kgf/cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>8</width>
       <height>8</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>P =pressÃ£o de trabalhol em Kgf/cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>8</width>
       <height>8</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>d = diametro interno em milimetros</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="tela_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>tela_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>bt_01</zorder>
   <zorder>tela_1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
